INSTALLING.rst

```diff
@@ -91,31 +91,28 @@ Tell CMake to search for packages in the virtual environment first::
 Install prerequisites
 ---------------------
 
-**HOOMD-blue** requires a number of libraries to build.
-
-- Required:
-
-  - C++11 capable compiler (tested with ``gcc`` 4.8, 5.5, 6.4, 7,
-    8, 9, ``clang`` 5, 6, 7, 8)
-  - Python >= 3.5
-  - NumPy >= 1.7
-  - pybind11 >= 2.2
-  - Eigen >= 3.2
-  - CMake >= 3.9
-  - For MPI parallel execution (required when ``ENABLE_MPI=on``):
-
+**HOOMD-blue** requires a number of libraries to build. The flags ``ENABLE_MPI``,
+``ENABLE_GPU``, ``ENABLE_TBB``, and ``BUILD_JIT`` each require additional libraries.
+
+General requirements
+    - C++11 capable compiler (tested with ``gcc`` 4.8, 5.5, 6.4, 7,
```

Comment:
These '-''s need to be in the first column. There also needs to be a blank line between `General requirements` and the start of the list. 
```suggestion

- C++11 capable compiler (tested with ``gcc`` 4.8, 5.5, 6.4, 7,
```